### DISCUSSION TITLE
Transfering of projects not working

### DISCUSSION THREAD

**Discussion by advany** [vercel/community#5240] [Account Issue] [answered]
### Question

I am unable to transfer a project.

When I click on transfer, it asks to what team, I select the team. Then it asks me if I want to transfer additional project (1 selected), I click next, then I see zero project will be transfered?

https://www.loom.com/share/369e33cd4af54e13b7f24658b8c95436?sid=abab5b45-7264-41b7-9011-7739056ba231

**Comment by swarnava** [comment, marked as answer]
Hi @advany, we have reached out to via email. Thank you!
